packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-email/main/go/summaryofsteps.md

```diff
@@ -1,7 +1,7 @@
-The sequence of steps for the sign-in with password and email flow is shown below.
+The following summarizes the steps involved in a sign-in flow with a password and email:
 
 <div class="full">
 
-![Displays a diagram of the sign-in flow using the email factor with the Golang sample application](/img/oie-embedded-sdk/oie-embedded-sdk-go-use-case-sign-on-email-factor.png)
+![Displays a sequence diagram of the sign-in with email and password flow](/img/oie-embedded-sdk/oie-embedded-go-sign-in-pwd-email-flow-diagram.png)
```

Comment:
Go Diagram comments:

1. Missing call to initlogin
2. If I remember Go conventions don't use parenthesis after method names
3. Missing HasStep call
4. Missing check email activity
5. Missing token storage activity

Reply:
1. Fixed
2. Good spot. I didn't know this convention. Have fixed.
3. Not going to include. This is just part of displaying the list of authenticators
4. See earlier
5. See earlier